tools/infrastructure/api_compare.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/env python
+
+import json
+import sys
+import argparse
+import xml.etree.ElementTree as ET
+from collections import namedtuple
+reload(sys)
+
+# Command line arguments parsing
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-f1", "--mobile_api_file",
+    type=str, default="MOBILE_API.xml",
+    help='Enter mobile api file name! ("MOBILE_API.xml" by default)'
+)
+
+parser.add_argument(
+    "-f2", "--hmi_api_file",
+    type=str, default="HMI_API.xml",
+    help='Enter hmi api file name! ("HMI_API.xml" by default)'
+)
+
+parser.add_argument(
+    "-o", "--output",
+    type=str, default="console",
+    choices=['xml', 'json', 'console'],
+    help='Choose report type! ("console" by default)'
+)
+
+args = parser.parse_args()
+
+# Enable Utf-8 support
+sys.setdefaultencoding('utf-8')
+
+mobile_api_tree = ET.parse(args.mobile_api_file)
```

Comment:
@AByzhynar ET name is a bit confusing, I need to look to the beginning of the file to see what is it.